be/src/olap/rowset/segment_v2/binary_plain_page.h

```diff
@@ -47,14 +47,13 @@ namespace segment_v2 {
 class BinaryPlainPageBuilder : public PageBuilder {
 public:
     BinaryPlainPageBuilder(const PageBuilderOptions& options)
-            : _size_estimate(0), _prepared_size(0), _options(options) {
+            : _size_estimate(0), _options(options) {
         reset();
     }
 
     bool is_page_full() override {
         // data_page_size is 0, do not limit the page size
-        return _options.data_page_size != 0 && (_size_estimate > _options.data_page_size ||
-                                                _prepared_size > _options.data_page_size);
+        return _options.data_page_size != 0 && (_size_estimate > _options.data_page_size);
```

Comment:
better
`return (_options.data_page_size != 0 && _size_estimate > _options.data_page_size);`

Reply:
why ？

Reply:
we might remove the `( )`
`return _options.data_page_size != 0 && _size_estimate > _options.data_page_size;`

or
`return (_options.data_page_size != 0) && (_size_estimate > _options.data_page_size);`